zzk/registry/registry.go

```diff
@@ -191,7 +190,7 @@ func (r *registryType) ensureKey(conn client.Connection, key string) error {
 		default:
 		}
 	}
-	return fmt.Errorf("could not create key: %s", key, err)
+	return nil
```

Comment:
I'm not sure this works - looks like we want to return an error if the timeout is reached.